app/routes/index.js

```diff
@@ -0,0 +1,7 @@
+import Ember from 'ember';
+
+export default Ember.Route.extend({
+  afterModel() {
```

Comment:
I'd use a `beforeModel()` hook here because we need the redirect to happen before it tries to grab a model - do you have a reason for using `afterModel()`?